script/sync_prod_to_gcp/sync_published.sh

```diff
@@ -1,4 +1,33 @@
 #!/bin/bash
+
+PIDFILE="/var/run/sync_to_gcp_cron_job.pid"
+
+if [ -f "$PIDFILE" ]; then
+    # Check if the process with the stored PID is still running
+    PID=$(cat "$PIDFILE")
+    if ps -p $PID > /dev/null; then
+        # Wait loop to wait until the lock is released
+        while ps -p $PID > /dev/null; do
+            sleep 1
```

Comment:
Maybe `sleep 5`?

Reply:
Thought about making it longer. The sync host is not overloaded as raison d'être is to run the syncing. I will make it to 5 regardless.